File: 9.cpp
#include <_ctype.h>
#include <string>
#include <vector>
using namespace std;

class Solution {
public:
  int isPalindrome(int x) {
    string number = to_string(x);
    int j = number.length() - 1;
    for (int i = 0; i < j; i++) {
      if (number[j] != number[i]) {
        return false;
      }
      j--;
    }
    return true;
  }
};

File: 9.ts
function isPalindrome(x: number): boolean {
  let numstring: string = x.toString();
  let j: number = numstring.length - 1;
  for (let i = 0; i < numstring.length; i++) {
    if (numstring[i] != numstring[j]) {
      return false;
    }
    if (i == j || i == j - 1) {
      return true;
    }
    j--;
  }
  return true;
}
